Additional edits to high-throughput screening section (#1025)
greenelab/covid19-review: content/20.pharmaceuticals.md

### High-Throughput Screening for Drug Repurposing

The drug development process is slow and costly and the development of compounds specifically targeted to an emerging viral threat is not a practical short-term solution.
Screening existing drug compounds for alternative indications is a popular alternative [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895; @doi:10.1039/D0CS01065K; @doi:10.1016/j.coi.2013.08.004].
HTS has been a goal of pharmaceutical development since at least the mid-1980s [@doi:10.1038/sj.bjp.0707373].
Traditionally, phenotypic screens were used to test which compounds would induce a desired change in _in vitro_ or _in vivo_ models, focusing on empirical, function-oriented exploration naïve to molecular mechanism [@doi:10.1126/scitranslmed.aab1201; @doi:10.1038/clpt.2012.236; @doi:10.1038/nrd4366].
In many cases, these screens utilize large libraries that encompass a diverse set of agents varying in many pharmacologically relevant properties (e.g., [@url:https://www.chembridge.com/screening_libraries/diversity_libraries]).
The compounds inducing a desired effect could then be followed up on.

> Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards emphasizing specific molecular targets that were hypothesized to have a desired effect and to screen for compounds interacting with the target.

Author: Yes! Much better!

### High-Throughput Analyses for Drug Repurposing

Because the drug development process is slow and costly and the development of compounds specifically targeted to an emerging viral threat are not a practical short-term solution, efforts to screen existing drug compounds for alternative indications have become popular [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895; @doi:10.1039/D0CS01065K; @doi:10.1016/j.coi.2013.08.004].
High-throughput screening has been a goal of pharmaceutical development since at least the mid-1980s [@doi:10.1038/sj.bjp.0707373].
Traditionally, phenotypic screens were used to test which compounds would induce a desired change in _in vitro_ or _in vivo_ models, focusing on empirical, function-oriented exploration naïve to molecular mechanism [@doi:10.1126/scitranslmed.aab1201; @doi:10.1038/clpt.2012.236; @doi:10.1038/nrd4366].

> Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards emphasizing specific molecular targets that were hypothesized to have a desired effect and to screen compounds for those interacting with the target.

Reviewer: ```suggestion
Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards emphasizing specific molecular targets that were hypothesized to have a desired effect and to screen for compounds interacting with the target.
```

> In the age of biological big data, _in silico_ studies have emerged as a new frontier [@doi:10.1016/j.coi.2013.08.004].

Reviewer: Are in silico screens necessarily tied to big data?  Something like an in silico screen that uses protein docking doesn't require big data to apply it to a new target (though the docking was likely developed and tuned on big data).

Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use computational HTS approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
Computational drug repurposing screens can take advantage of big data in biology [@doi:10.1038/nrd.2018.168] and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
Advancements in robotics also facilitate the experimental component of HTS [@doi:10.1039/D0CS01065K].
For viral diseases, the goal of drug repurposing is typically to identify existing drugs that have an antiviral effect likely to impede the virus of interest.
While both small molecules and biologics can be candidates for repurposing, the significantly lower price of many small molecule drugs means that they are typically more appealing candidates [@doi:10.1089/adt.2019.950].
Depending on the design, screens can be either hypothesis driven or hypothesis free.

> #### Hypothesis-Driven Screening

Reviewer: Above we introduced the distinction between target- versus phenotypic-based screens.  Is target-based equivalent to hypothesis-driven, or is hypothesis-driven versus hypothesis-free a separate classification?

Reviewer 2: I think that phenotypic screens could be hypothesis-guided or hypothesis-free depending on the library of compounds used. I agree with @rando2's interpretation above though - mostly as broad as possible.

Author: I think with my latest commit (0a02366a), I've (hopefully) moved things around enough to keep the focus on computational screens, with the context from the target-based vs phenotypic screen debate brought in just to highlight the importance of unifying hypothesis-based and discovery-driven methods.

In the case of hypothesis-driven screens, there are several properties that might identify a compound as a candidate for an emerging viral disease. 
Drugs that interact with a target that is shared between pathogens or between a viral pathogen and another illness (i.e., a cancer drug with antiviral potential) are potential candidates, as are drugs that are thought to interact with additional molecular targets beyond those they were developed for [@doi:10.1016/j.tim.2018.04.004].
This research can be driven by _in vitro_ or _in silico_ experimentation.

> For computational analyses, such approaches require identifying compounds that modulate pre-selected proteins in the virus or host, and as a result, they build on experimental research characterizing the molecular features of the virus, host, and candidate compounds [@doi:10.1016/j.coi.2013.08.004].

Reviewer: ```suggestion
For computational analyses, such approaches require identifying compounds that modulate pre-selected proteins in the virus or host.
As a result, they build on experimental research characterizing the molecular features of the virus, host, and candidate compounds [@doi:10.1016/j.coi.2013.08.004].
```

Other M^Pro^ inhibitors are also being evaluated in clinical trials [@clinicaltrials:NCT04535167; @clinicaltrials:NCT04960202; @clinicaltrials:NCT04960202].
Pending the results of clinical trials, N3 remains a computationally interesting compound based on both computational and experimental data, but whether these potential effects will translate to the clinic remains unknown.

#### Hypothesis-Free Screening

> Hypothesis-free drug screening began twenty years ago with the testing of libraries of drugs experimentally and today, like many other areas of biology, _in silico_ analyses have become increasingly popular and feasible through advances in biological big data [@doi:10.1021/acs.jcim.0c00861; @doi:10.1080/23808993.2019.1617632].

Reviewer: ```suggestion
Hypothesis-free drug screening began twenty years ago with the testing of libraries of drugs experimentally.
Today, like many other areas of biology, _in silico_ analyses have become increasingly popular and feasible through advances in biological big data [@doi:10.1021/acs.jcim.0c00861; @doi:10.1080/23808993.2019.1617632].
```

Many efforts have collected data about interactions between drugs and SARS-CoV-2 and about the host genomic response to SARS-CoV-2 exposure, allowing for hypothesis-free computational screens that seek to identify new candidate therapeutics.
Thus, they utilize a systems biology paradigm to extrapolate the effect of a drug against a virus based on the host interactions with both the virus and the drug [@doi:10.1016/j.coi.2013.08.004].

> Resource such as the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090], facilitate such discovery-driven approaches.

Reviewer: ```suggestion
Resources such as the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090], facilitate such discovery-driven approaches.
```

#### Potential and Limitations of High-Throughput Analyses

These examples illustrate that HTS-based evaluations of drug repurposing potential can provide some valuable insights, but there are significant limitations to these methods.
The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].
As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.

> Additionally, much of the research used to identify and validate targets is not reproducible [@doi:10.1038/nrd3439-c1], presenting a possible limitation to target-based experimental screening and both hypothesis-driven and hypothesis-free computational HTS efforts. 

Reviewer: This sentence now makes me think that hypothesis-driven and hypothesis-free only apply to computational screens?

I only skimmed the reference, but there seems to be more nuance to its results.  There is some debate about whether inconsistencies between published data and similar follow up experiments means the results are irreproducible.

Some efforts to repurpose antivirals have focused on host, rather than viral, proteins [@doi:10.1016/j.coi.2013.08.004], which might be expected to translate poorly _in vivo_ if the targeted proteins serve essential functions in the host.

> Furthermore, it's not clear to what extent large-scale computational HTS studies, especially those that are hypothesis free, produce insights that wouldn't be gained from smaller experimental screening efforts [@doi:10.1039/D0CS01065K; @doi:10.1126/science.abj9488].

Reviewer: Does this primarily apply to computational _repurposing_ screens?  We don't talk about it, but computational methods that assess a billion synthesizable compounds are looking at a much different part of the chemical space than small experimental screens.

Author: I think I'm a little out of my depth here. I might remove this sentence because I don't think it provides anything super critical to making the point!
```suggestion
```

It is possible that fluvoxamine also exerts an antiviral effect [@doi:10.3389/fphar.2021.652688].
Thus, Sigma-1 receptor activity may contribute to fluvoxamine's potential effects in treating COVID-19, but is not the only mechanism by which this drug can interfere with disease progression.

#### Potential and Limitations of High-Throughput Analyses

Computational screening allows for a large number of compounds to be evaluated to identify those most likely to display a desired behavior or function.
This approach can be guided by a hypothesis or can aim to discover underlying characteristics that produce new hypotheses about the relationship between a host, a virus, and candidate pharmaceuticals.
The examples outlined above illustrate that HTS-based evaluations of drug repurposing can potentially provide valuable insights. 
Computational techniques were used to design compounds targeting M^Pro^ based on an understanding of how this protease aids viral replication, and M^Pro^ inhibitors remain promising candidates [@doi:10.1039/D0CS01065K], although the clinical trial data is not yet available.

> Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest, and although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.

Reviewer: ```suggestion
Similarly, computational analysis correctly identified the Sigma-1 receptor as a protein of interest.
Although the process of identifying which drugs might modulate the interaction led to an emphasis on candidates that ultimately have not been supported, fluvoxamine remains an appealing candidate.
```

The difference between the preliminary evidence for fluvoxamine compared to other drugs that interact with Sigma receptors underscores a major critique of hypothesis-free HTS in particular: while these approaches allow for brute force comparison of a large number of compounds against a virus of interest, they lose the element of expertise that is associated with most successes in drug repurposing [@doi:10.1021/acs.jcim.0c00861].

There are also practical limitations to these methods.
One concern is that computational analyses inherently depend on the quality of the data being evaluated.
The urgency of the COVID-19 pandemic led many research groups to pivot towards computational HTS research without familiarity with best practices in this area [@doi:10.1039/D0CS01065K].

> As a result, there is an excessive amount of information available from computational studies [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], but not all of it is high-quality.

Reviewer: This manual reference isn't working. I'll look into it.

Author: Thank you Tony!

Reviewer: b9ca24f fixed this

Additionally, the literature used to identify and validate targets can be difficult to reproduce [@doi:10.1038/nrd3439-c1], which may pose challenges to target-based experimental screening and to _in silico_ screens. 
Some efforts to repurpose antivirals have focused on host, rather than viral, proteins [@doi:10.1016/j.coi.2013.08.004], which might be expected to translate poorly _in vivo_ if the targeted proteins serve essential functions in the host.

> Concerns about the practicality of hypothesis-free screens to gain novel insights is underscored by the fact that very few or possibly no success stories have emerged from hypothesis-free screens over the past twenty years [@doi:10.1021/acs.jcim.0c00861].

Reviewer: ```suggestion
Concerns about the practicality of hypothesis-free screens to gain novel insights are underscored by the fact that very few or possibly no success stories have emerged from hypothesis-free screens over the past twenty years [@doi:10.1021/acs.jcim.0c00861].
```

These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA [@doi:10.1038/nrd4366].
Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
This shift highlights the importance of prioritizing candidates that are likely to have a desired effect beyond computational or experimental models.
While this is easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.
Data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].

The strategies outlined here for drug repurposing reflect the same patterns found in HTS.

> In COVID-19, candidates for repurposing have been identified through both phenotypic (clinically indicated) and target-based approaches.

Reviewer: What is meant by "phenotypic (clinically indicated)"?

Author: This hopefully removes some of the convolution!
```suggestion
In COVID-19, candidates for repurposing have been identified through target-based approaches, but some of the most successful therapeutics have been established medications applied off-label.
```

Drugs like dexamethasone have a known effect on the human body, and in the case of COVID-19, this effect was found to be beneficial to patient outcomes.
Attempts to target the virus or host-virus interactions have produced mixed results in the clinic.
Remdesivir, which has emerged as a success for the target-based approach to identifying candidate COVID-19 therapeutics for repurposing, was first identified in a hypothesis-driven HTS effort to find compounds active against RNA viruses _in vitro_ [@doi:10.1021/acs.jmedchem.6b01594].
Promising preclinical results translated poorly to an initial RCT for Ebola [@doi:10.7573/dic.2020-4-14], although the results have been more promising for COVID-19 (Table @tbl:summary-table).

> Favipiravir, which uses a similar mechanism, has not been supported as a therapeutic for COVID-19, and HCQ, which was proposed to have antiviral effects relevant to the treatment of COVID-19 based on a mechanism that is known to translate poorly _in vivo_, was found to be harmful to patients.

Reviewer: Where does HCQ fit in this summary paragraph?  It follows two target-based examples, but is it considered target or phenotypic based?

Author: I think drop HCQ, it muddies the point a bit.
```suggestion
Favipiravir, which uses a similar mechanism, has not been supported as a therapeutic for COVID-19.
```

> The unpredictable nature of successes and failures of mechanism-based drug repurposing thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propogate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].

Reviewer: "mechanism-based drug repurposing" is yet another new term that could confuse the existing classifications described.

Author: ```suggestion
The unpredictable nature of successes and failures of drug repurposing thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
```

> 

Reviewer: May want to change "mechanism of action" below to MMOA.

Despite the fact that apparently mechanistically relevant drugs may exist, identifying effective treatments for a new viral disease is extremely challenging.
Targets of repurposed drugs are often non-specific, meaning that the MMOA can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
The difference in the current status of remdesivir and favipiravir as treatments for COVID-19 (Table @tbl:summary-table) underscores how difficult to predict whether a specific compound will produce a desired effect, even when the mechanisms are similar.
Furthermore, the fact that many candidate COVID-19 therapeutics were ultimately identified because of their shared propensity to induce phospholipidosis underscores how challenging it can be to identify a mechanism _in silico_ or _in vitro_ that will translate to a successful treatment.

> While significant progress has been made in the first 18 months of the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.

Reviewer: "first 18 months" may be unclear because the start data is not precise.

Author: ```suggestion
While significant progress has been made thus far in the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.
```
I want to say "year plus" but that probably isn't formal enough!
